### PR TITLE
Describe conventions used in CRX docs

### PR DESCRIPTION
**Pull request by dotproto**
Adds a new page to the Handbook that describes the conventions used in the Chrome Extensions and Chrome Web Store documentation sets.

[Preview page](https://deploy-preview-2071--developer-chrome-com.netlify.app/docs/handbook/extensions/)

### CODE REVIEW

**Comment by dotproto** [comment]
Requesting reviews from @jpmedley for editing and @devnook for this use of the handbook.

**Comment by AmySteam** [comment]
@dotproto This looks great! I was wondering if we should also mention the metadata available for the extensions docs, as opposed to the blog entries. For instance we don't have tags and author. Just a thought 🤔, but it also might be out of scope.

File: site/en/docs/handbook/extensions/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+## Link conventions {: #links }
+
+### Footer links
+
+By convention, extensions docs strongly prefer named footer links over inline links. Named links
+have a couple of advantages over inline links.
+
+- **Named links are typically shorter.** This makes it much easier to read and
+work with source Markdown file.
```
Comment (jpmedley) [inline site/en/docs/handbook/extensions/index.md:45]:
```suggestion
work with the source markdown file. 
```

File: site/en/docs/handbook/extensions/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+congue lacus aliquet quis.
+
+[mdn-global]: https://developer.mozilla.org/en-US/docs/Glossary/Global_object
+
+// Don't do
+Lorem ipsum dolor [sit
+amet](https://developer.mozilla.org/en-US/docs/Glossary/Global_object),
+consectetur adipiscing elit. Quisque aliquam rutrum pellentesque. Ut
+tincidunt purus ex, eget congue lacus aliquet quis.
+```
```
Comment (jpmedley) [inline site/en/docs/handbook/extensions/index.md:62]: Compare widget.

File: site/en/docs/handbook/extensions/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+### Internal links {: #internal-links }
+
+When linking to other resources on `developer.chrome.com`, use the absolute path of the page rather
```
Comment (jpmedley) [inline site/en/docs/handbook/extensions/index.md:66]: Is this a handbook for our writers or external extension writers? This is not only contrary to our handbook, we also have a ci test that flags it in PRs.

File: site/en/docs/handbook/extensions/index.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+than a relative path or full URL.
+
+```md
+// Do
+[doc-page]: /docs/extensions/page/
+
+// Don't do
+[doc-page]: https://develper.chrome.com/docs/extensions/page/
+[doc-page]: page/
+```
```
Comment (jpmedley) [inline site/en/docs/handbook/extensions/index.md:76]: Compare widget.

File: site/en/docs/handbook/extensions/index.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+```md
+// Do
+[doc-page]: /docs/extensions/dir/page/
+[doc-page]: /docs/extensions/sub/dir/page/#header
+
+// Don't do
+[doc-page]: /docs/extensions/dir/page
+[doc-page]: /docs/extensions/sub/dir/page#header
+```
```
Comment (jpmedley) [inline site/en/docs/handbook/extensions/index.md:88]: Compare widget.

File: site/en/docs/handbook/extensions/index.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+## Code formatting {: #code-formatting }
+
+Code samples in extensions documentation should be formatted using the Chromium project's `git cl
+format` command. To use this command, you must have [depot_tools][depot-tools] installed. See [Using clang-format][clang-format] for additional details.
```
Comment (jpmedley) [inline site/en/docs/handbook/extensions/index.md:93]:
```suggestion
format` command. To use this command, you must first install [depot_tools][depot-tools]. See [Using clang-format][clang-format] for additional details.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by dotproto** [comment]
Thanks for [the suggestion](https://github.com/GoogleChrome/developer.chrome.com/pull/2071#issuecomment-1029185306), @AmySteam. I've added a Metadata section to the top of the page.

**Review by jpmedley — CHANGES_REQUESTED** [review]
Apologies. I started reading before I realized I had reviewed this already. I found an item that must be fixed and one that should be.

File: site/en/docs/handbook/extensions/index.md (hunk 3 above)
Comment (dotproto) [reply site/en/docs/handbook/extensions/index.md]: Commit 39cc94f clarifies the pathing guidance. I think we can resolve this conversation.

**Comment by dotproto** [comment]
@jpmedley, I think all outstanding issues have been addressed. Please take another look when you can.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**